jupyterlite_pyodide_kernel/_version.py

```diff
@@ -1,2 +1,2 @@
 """source of truth for ``jupyterlite-pyodide-kernel``` version."""
-__version__ = "0.1.1"
+__version__ = "0.2.0a0"
```

Comment:
Bumping the version is done when making the release with the releaser so we can revert that changes and other bumps in the `package.json`: https://github.com/jupyterlite/pyodide-kernel/blob/main/RELEASE.md#specifying-a-version-spec

Reply:
yep, i know: see above. I wanted to actually see something working with the interplay between the new versions.

and the `check-release` is showing it can handle going up to `0.2.0a1`.

Reply:
Not sure to understand what bumping the version in this PR helps with?

Reply:
It is for _my_ understanding, and presumably `lerna/yarn/esbuild/webpack`'s, as i was having issues with cache collisions, pulling the old packages, etc. even after blowing everything away, new envs all that.

also this leaves accurately-numbered wheels/sdist in the RTD build, as i'm trying to do downstream testing as well, due to the limited surface in this test suite and docs site.

and again, the bumper does fine on check-release, correctly going to `0.2.0a1`, so won't hurt anybody.

Reply:
> also this leaves accurately-numbered wheels/sdist in the RTD build

Are they not already "scoped" per sub-domain, for example https://jupyterlite-pyodide-kernel--58.org.readthedocs.build

Reply:
Should the version bump be reverted if downstream testing is done now?